Resources/views/Collector/db.html.twig

```diff
@@ -107,18 +147,18 @@
                                 <img alt="-" src="data:image/gif;base64,R0lGODlhEgASAMQSANft94TG57Hb8GS44ez1+mC24IvK6ePx+Wa44dXs92+942e54o3L6W2844/M6dnu+P/+/l614P///wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAABIALAAAAAASABIAQAVCoCQBTBOd6Kk4gJhGBCTPxysJb44K0qD/ER/wlxjmisZkMqBEBW5NHrMZmVKvv9hMVsO+hE0EoNAstEYGxG9heIhCADs=" style="display: none;" />
                                 <span style="display: none">Shrink query</span>
                                 <span id="smallcode-{{ i }}-{{ loop.parent.loop.index }}">
-                                    {{ query.sql|doctrine_minify_query|raw }}
+                                    {{ query.sql|doctrine_minify_query }}
                                 </span>
                             </div>
-                            <code id="code-{{ i }}-{{ loop.parent.loop.index }}">
-                                {{ query.sql|doctrine_pretty_query(i, loop.parent.loop.index)|raw }}
-                            </code>
+                            <div id="code-{{ i }}-{{ loop.parent.loop.index }}" style="display: none;">
+                                {{ query.sql|doctrine_pretty_query(i, loop.parent.loop.index) }}
+                            </div>
                             <span id="original-query-{{ i }}-{{ loop.parent.loop.index }}" style="display: none;">
-                                {{ query.sql|doctrine_replace_query_parameters(query.params)|raw }}
+                                {{ query.sql|doctrine_replace_query_parameters(query.params) }}
```

Comment:
your diff seems to include many changes which should not be in this PR (they are already in master). Can you rebase your branch to fix this ?